Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -3621,11 +3653,22 @@ private unsafe void OnUpdate(InputUpdateType updateType, ref InputEventBuffer ev
                         break;
                 }
 
-                m_Metrics.totalEventProcessingTime +=
+                ResetCurrentProcessedEventBytesForDevices();
+
+                // Update metrics (exposed via analytics and debugger)
+                var eventProcessingTime =
                     ((double)(Stopwatch.GetTimestamp() - processingStartTime)) / Stopwatch.Frequency;
+                m_Metrics.totalEventCount += totalEventCount;
+                m_Metrics.totalEventBytes += totalEventSizeBytes;
+                m_Metrics.totalEventProcessingTime += eventProcessingTime;
                 m_Metrics.totalEventLagTime += totalEventLag;
 
-                ResetCurrentProcessedEventBytesForDevices();
+                // Profiler counters
```

Comment:
I'd add a comment that these counters are PerFrame, as we can call update more than once?
Or Maybe a prefix of something like `InputStatistics.EventCountPerFrame.Value` or `PerUpdate`, something like that.

Also if could we use `ProfileRecorder` for this?